borg/assembl_borg_backup.sh.jinja2

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+#You need to set these two environment variables, for example:
+#ASSEMBL_PATH=/home/benoitg/development/assembl
+#REPOSITORY=www-data@coeus.ca:/media/backup/assembl_backups.borg
+ASSEMBL_PATH = {{assembl_path}}
+REPOSITORY = {{repository}}
+set -x BORG_RELOCATED_REPO_ACCESS_IS_OK=yes
+BORG_PASSPHRASE= {{borg_passphrase}} borg init --encryption=keyfile $REPOSITORY || true
+echo "Do not worry if the above command fails, it is expected to fail except the first time it is run"
+
+cd $ASSEMBL_PATH
+#In case the database backup fails
+$ASSEMBL_PATH/venv/bin/assembl-db-manage local.ini backup
+#Make sure we back up the database dump from the last deployment:
+cp --dereference $ASSEMBL_PATH/assembl-backup.pgdump $ASSEMBL_PATH/assembl-backup-real.pgdump
+NAME="`hostname`-`basename $ASSEMBL_PATH`-`date --iso-8601='minutes'`"
+#set -x
+borg create \
+    $REPOSITORY::$NAME \
+    $ASSEMBL_PATH \
```

Comment:
You should also add a path to an RC file, where piwik backups will go to on the server, and add that path in the template here. That's part of the reason for moving this document to a template :)